src/components/Link/Link.mdx

```diff
@@ -47,6 +47,11 @@ the relation names "nofollow", "noopener" and "noreferrer".
 
 <Canvas of={LinkStories.Accessibility} />
 
+The `Link` component also has a `screenreaderOnlyText` prop that can be used to
+add additional text to the link that is only visible to screen readers. This can
+be used to provide additional context to the link when the text is short, such
+as visible "Read more..." text.
```

Comment:
This should also be added to the story for `Button`.

Reply:
A similar paragraph or the example? There is a similar paragraph already but I'll add the example.

Reply:
I was referring to the paragraph, but I must have missed it.  Sorry.  And I see you adjusted it slightly in this update.